api/managementpb/agent.proto

```diff
@@ -0,0 +1,116 @@
+syntax = "proto3";
+
+package management;
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "api/managementpb;managementpb";
+
+message UniversalAgent {
```

Comment:
> most APIs usually returns only main fields in List endpoints and have separate Get endpoint to return all fields related to the entity.

That was exactly the goal - we need all fields delivered by this endpoint.

Reply:
Note. We agreed to refactor this endpoint so that:
1/ it only outputs the attributes that FE requires to populate the list of agents in table view
2/ create a `/v1/managemenet/Agent/Get` endpoint so that FE can get all attributes when expanding the agent row.